src/Bridge/Symfony/Bundle/Resources/views/SwaggerUi/index.html.twig

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>{{ title }} - API Platform</title>
+
+    <link rel="icon" type="image/png" href="{{ asset('bundles/apiplatform/swagger-ui/images/favicon-32x32.png') }}" sizes="32x32">
+    <link rel="icon" type="image/png" href="{{ asset('bundles/apiplatform/swagger-ui/images/favicon-16x16.png') }}" sizes="16x16">
+    <link href="{{ asset('bundles/apiplatform/swagger-ui/css/typography.css') }}" media="screen" rel="stylesheet">
+    <link href="{{ asset('bundles/apiplatform/swagger-ui/css/reset.css') }}" media="screen" rel="stylesheet">
+    <link href="{{ asset('bundles/apiplatform/swagger-ui/css/screen.css') }}" media="screen" rel="stylesheet">
+    <link href="{{ asset('bundles/apiplatform/swagger-ui/css/reset.css') }}" media="print" rel="stylesheet">
+    <link href="{{ asset('bundles/apiplatform/swagger-ui/css/print.css') }}" media="print" rel="stylesheet">
+    <script src="{{ asset('bundles/apiplatform/swagger-ui/lib/jquery-1.8.0.min.js') }}"></script>
+    <script src="{{ asset('bundles/apiplatform/swagger-ui/lib/jquery.slideto.min.js') }}"></script>
+    <script src="{{ asset('bundles/apiplatform/swagger-ui/lib/jquery.wiggle.min.js') }}"></script>
+    <script src="{{ asset('bundles/apiplatform/swagger-ui/lib/jquery.ba-bbq.min.js') }}"></script>
+    <script src="{{ asset('bundles/apiplatform/swagger-ui/lib/handlebars-2.0.0.js') }}"></script>
+    <script src="{{ asset('bundles/apiplatform/swagger-ui/lib/underscore-min.js') }}"></script>
+    <script src="{{ asset('bundles/apiplatform/swagger-ui/lib/backbone-min.js') }}"></script>
+    <script src="{{ asset('bundles/apiplatform/swagger-ui/swagger-ui.min.js') }}"></script>
```

Comment:
I prefer to include the dist swagger ui (it will ease upgrades). I think that the best solution is to open a PR on the Swagger UI repository to fix their build process.